src/themes/default/components/core/ProductTile.vue

```diff
@@ -202,7 +202,10 @@ $color-white: color(white);
   &__thumb {
     padding-bottom: calc(143.88% / (164.5 / 100));
     @media screen and (min-width: 768px) {
-      padding-bottom: calc(300% / (276.5 / 100));
+      padding-bottom: calc(250% / (276.5 / 100));
+    }
+    @media screen and (min-width: 1024px) {
+      padding-bottom: calc(200% / (276.5 / 100));
```

Comment:
it's fine, please next time maybe try to refactor though, because calc is quite inefficient, sometimes breaks the FPS